embc-app/embc-app.csproj

```diff
@@ -34,6 +34,8 @@
     <PackageReference Include="Microsoft.AspNetCore.SpaServices.Extensions" Version="2.1.1" />
 
     <PackageReference Include="Microsoft.Extensions.Diagnostics.HealthChecks.EntityFrameworkCore" Version="2.2.0" />
+
+    <PackageReference Include="Microsoft.VisualStudio.Web.CodeGeneration.Design" Version="2.1.5" />
```

Comment:
What's that package do?

Reply:
Don't know where it came from, will revert that for sure.

Reply:
Removed